specification/policyinsights/resource-manager/Microsoft.PolicyInsights/stable/2019-10-01/operations.json

```diff
@@ -0,0 +1,150 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Policy Insights Operations API",
+    "version": "2019-10-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
```

Comment:
>securityDefinitions [](http://example.com/codeflow?start=3&length=19)

please address the ~[Staging] Swagger LintDiff errors here [https://github.com/Azure/azure-rest-api-specs/pull/20802/checks?check_run_id=8975108203](https://github.com/Azure/azure-rest-api-specs/pull/20802/checks?check_run_id=8975108203)